README.md

```diff
@@ -22,13 +22,13 @@ To get started, it is best to have the latest JDK and Gradle installed. The HEAD
 
 #### Building from the command line
 
-Official builds are currently using with JDK 8, even though the `core` module is compatible with JDK 7 and later.
+Official builds are currently using JDK 11, even though the `core` module is compatible with JDK 8 and later.
```

Comment:
I wonder if we should perhaps remove this line? It confuses me because I think this is the build requirement, but actually it seems to describe the API requirement, which is already described in the paragraph above.

Reply:
I agree. I'll remove it and squash when I'm next at my computer.